docs/compose/.env

```diff
@@ -1,3 +1,5 @@
+# WARNING: this file is being deprecated over the new setup utility, found at https://setup.mailu.io
```

Comment:
Okay, never knew about this workflow honestly.

So, some time ago @kaiyou  took the design decision that we should move to a setup utility, before we released 1.6. This is to deal with more complex deployment demands from our users. It would become too much of a hassle to include all possible deployment flavors in a single `docker-compose.yml` and `.env`. I mean, running an empty container like `/mailu/none` just to keep the syntax valid?

I've seen that, even after releasing the setup utility people are still doing their additions to the "old" `.env`. Adding this warning hopefully will push development efforts to the setup utility, which until this moment is (almost) only maintained by me and @ionutfilip. We don't intend on maintaining this all by ourselves.

Back to the testing workflow:
- Creating updated `docker-compose.yml` and `mailu.env` using setup only takes a minute. You don't have to modify all options. Just select the services you need, write in the bind addresses, mail domain and the hostname and that's it. The rest of the defaults are fine.
- Setup gives you two `wget` commands which can be copied-paste into the terminal as-is.

I don't intend to remove `.env` and `docker-compose.yml` anywhere soon, but I also don't intend to maintain it with the latest design decisions. Eg: unbound resolver option is not there.

Reply:
Of course setup can be improved a lot, btw. For instance, instead of documenting Traefik, we could support it as an option in the setup instead. Perhaps we can also store the user settings to a cookie and pre-fill the form fields when a user comes back to the setup page. So that he does not need to start from scratch every time.

Reply:
The latter is exactly my point: If I — as a enduser/admin — want to update my already existing deployment to a newer version using the setup utility, then I have to carefully copy-paste all my settings into the setup utility. Some settings currently aren’t represented, but that’s not a blocker, just required enhancements.

On the other hand, I find managing such a changeset, where there was one initial version (user used setuptool 1st time), then manual changes (he changed some thins manually because it was easier than re-entering stuff in setup-tool, or because it’s not supported yet), then there’s a new version which potentillly added some options and removed some options, much easier using merging-tools in plain text. The alternative is the user clicking through setup again, receiving new files, then diffing them and needing to decide whether a diff is due to mis-click in the setup, updated results from setup, or one of his/her prior manual changes.

It’s not like I want to block this harshly right now … just discussing how to please the users best. And for first-time users and simpler setups I agree: The setup tool is great for that!

Reply:
Okay, I promise I won't delete the files ;). But be aware that some latest additions to Mailu are not there. (Postgresql, Unbound etc.)

Meanwhile, can I have your approval on this PR? I kinda need it merged for some other jobs.

Reply:
If they’re not regularly updated, they’re as useless as when being deleted for the described usecase :grinning: .

If the thought of maybe some day (~soon~ish) thinking about a easier user-setup-updating workflow has been planted in a few heads, then I’m fine with this in any case.

Thanks for the convenient setup-tool btw … It’s not as if I dislike it entirely or something, in fact it’s quite cool! So, please don’t misunderstand my urge to discuss ^_^.